enterprise_access/apps/content_assignments/constants.py

```diff
@@ -104,7 +104,7 @@ class AssignmentAutomaticExpiredReason:
     """
     Reason for assignment automatic expiry.
     """
-    NIENTY_DAYS_PASSED = 'NIENTY_DAYS_PASSED'
+    NINETY_DAYS_PASSED = 'NINETY_DAYS_PASSED'
```

Comment:
Noticed a typo in this constant, refactored all cases where its used with the correct spelling for the value and variable